src/Browscap/Data/Factory/DivisionFactory.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * This file is part of the browscap package.
```

Comment:
Don't need copyright notices everywhere; it's already explained in `LICENSE`

Reply:
done